include/ofi_shm.h

```diff
@@ -88,12 +88,22 @@ enum {
 #define SMR_RX_COMPLETION	(1 << 3)
 #define SMR_MULTI_RECV		(1 << 4)
 
-/* CMA capability */
-enum {
-	SMR_CMA_CAP_NA,
-	SMR_CMA_CAP_ON,
-	SMR_CMA_CAP_OFF,
-};
+/*
+ * CMA capability
+ * Low 4 bits for self cap; high 4 bits for peer cap;
+ * In general:
+ * smr_self_cma = smr->cma_cap & SMR_CMA_PEER_MASK
+ * smr_peer_cma = smr->cma_cap & SMR_CMA_SELF_MASK
+ */
+#define SMR_CMA_NA	(0)
+#define SMR_CMA_ON	(1 << 0)
+#define SMR_CMA_OFF	(1 << 1)
```

Comment:
I'm not sure we really even need a tri-state here.  Just true/false seems sufficient, with a default of false.

Reply:
The problem here is that we need to know if the endpoint has done CMA detection or not. 
Having a default of false dose not tell us if CMA is not permitted or CMA detection has not been done yet.